lib/routes/account.js

```diff
@@ -52,6 +52,7 @@ module.exports = function (
         var authSalt = crypto.randomBytes(32)
         var authPW = Buffer(form.authPW, 'hex')
         var locale = request.app.acceptLanguage
+        var userAgent = request.headers['user-agent']
```

Comment:
I was also chatting with  @ckarlof about this, and he suggested that we log only some subset of the user-agent information.  Keeping too much of it could easily stray into creepy-tracking territory, but pulling out the high-level fields like browser, OS and platform would not.

The sync client record fields are probably a good starting point for what level of information to keep: https://docs.services.mozilla.com/sync/objectformats.html#clients